docs/src/main/asciidoc/security-getting-started.adoc

```diff
@@ -507,8 +506,42 @@ public class User extends PanacheEntity {
 
 @Entity
 public class Role extends PanacheEntity {
+ follow the instructions in the next sections and create the application step by step.
```

Comment:
hi @michelle-purcell Sorry if I've missed something but looks like this section been duplicated 3 times. may be it is a github rebase issue ?

Reply:
Hi @michelle-purcell no problems at all :-), please follow up with another PR whenever you are ready, ping me please if some clarifications are needed